.github/workflows/main.yml

```
@@ -216,6 +216,13 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      # patch the sources list to use the default ubuntu repository (to prevent azure giving us packages...)
+      - name: Fix apt sources.list (prevents downtime from azure repositories not being reliable)
```

The comment should define the purpose, and the action name should define what's being done.

```suggestion
      # Azure repositories are not reliable, we need to prevent azure giving us packages.
      - name: Make apt sources.list use the default Ubuntu repositories.
```